helpers/deployContracts.ts

```diff
@@ -1,31 +1,277 @@
 import * as aragonContracts from "@aragon/core-contracts-ethers";
-import {ethers} from "ethers";
-import {Server} from 'ganache'
+import {ethers, Signer, BigNumberish, Contract} from "ethers";
 
-export async function deploy(server: Server) {
-  const accounts = await server.provider.getInitialAccounts()
+import ENSRegistry from "@ensdomains/ens-contracts/artifacts/contracts/registry/ENSRegistry.sol/ENSRegistry.json";
+import PublicResolver from "@ensdomains/ens-contracts/artifacts/contracts/resolvers/PublicResolver.sol/PublicResolver.json";
+
+const WALLET_ADDRESS = "0x8626f6940E2eb28930eFb4CeF49B2d1F2C9C1199";
+
+export interface Deployment {
+  daoFactory: aragonContracts.DAOFactory;
+  erc20Repo: aragonContracts.PluginRepo;
+  erc20PluginSetup: aragonContracts.ERC20VotingSetup;
+  allowListRepo: aragonContracts.PluginRepo;
+  allowListPluginSetup: aragonContracts.AllowlistVotingSetup;
+}
+
+export async function deploy(): Promise<Deployment> {
   const provider = new ethers.providers.JsonRpcProvider(
     "http://127.0.0.1:8545"
   );
-  const owner = new ethers.Wallet(accounts[Object.keys(accounts)[0]].secretKey, provider);
-
-  // @ts-ignore
-  const tokenFactory = new aragonContracts.TokenFactory__factory();
-  const token = await tokenFactory.connect(owner).deploy();
-  // @ts-ignore
-  const registryFactory = new aragonContracts.Registry__factory();
-  const registry = await registryFactory.connect(owner).deploy();
-  // @ts-ignore
-  const daoFactory = new aragonContracts.DAOFactory__factory();
-  const dao = await daoFactory
-    .connect(owner)
-    .deploy(registry.address, token.address);
-
-  // send ETH to hardcoded wallet in tests
-  await owner.sendTransaction({
-    to: "0x8626f6940E2eb28930eFb4CeF49B2d1F2C9C1199",
-    value: ethers.utils.parseEther("50.0"),
-  });
-
-  return dao;
+  const owner = provider.getSigner();
```

Comment:
```suggestion
  const deployWallet = provider.getSigner();
```

Reply:
renamed to `deployOwnerWallet` because it is also the owner of some contracts